NitroxServer/Serialization/World/WorldPersistence.cs

```diff
@@ -229,7 +231,10 @@ private World CreateFreshWorld()
                 Log.Info("Server has no password set.");
             }
             Log.InfoSensitive("Admin Password: {password}", config.AdminPassword);
-            Log.Info("To get help for commands, run {commandName} in console or {chatCommandName} in chatbox", "help", "/help");
+            Log.Info($"Admin Password: {config.AdminPassword}");
```

Comment:
Is this "double-call" correct?

Reply:
again likely a merge mistake, its removed now